pkg/mcpfile/parser_test.go

```diff
@@ -29,10 +30,10 @@ func TestParseMcpFile(t *testing.T) {
 						Name:    "test-server",
 						Version: "1.0.0",
 						Runtime: &ServerRuntime{
-							TransportProtocol: TransportProtocolStreamableHttp,
+							TransportProtocol: "streamablehttp",
```

Comment:
interesting they do not have constant for that.

since we use this string a couple of times, let's - for now - have our own